contracts/math/SafeMath.sol

```diff
@@ -45,4 +45,17 @@ library SafeMath {
     assert(c >= a);
     return c;
   }
+
+  /**
+  * @dev Integer modulo operation of two numbers.
+  */
+  function mod(uint256 a, uint256 b) internal pure returns (uint256 c) {
+    // d = a / b
+    // a = d * b + c
```

Comment:
Consider adding a small comment here noting that truncation happens when `d` is calculated. Otherwise it may be confusing, since for a real number `d`, `a = d * b + c = (a / b) * b + c = a + c`, which obviously doesn't hold.